src/Symfony/Component/Form/Form.php

```diff
@@ -766,11 +766,7 @@ public function isValid()
             return true;
         }
 
-        if (count($this->getErrors(true)) > 0) {
-            return false;
-        }
-
-        return true;
+        return 0 === count($this->getErrors(true));
```

Comment:
because being explicit is always better. We try to avoid using `empty` when possible.